src/index.html

```diff
@@ -11,7 +11,133 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form-group">
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input
+          type="text"
+          name="surname"
+          autocomplete="off">
+        </label>
+        <label class="form-field">
+          Name:
+          <input
+          type="text"
+          name="name"
+          autocomplete="off">
+        </label>
+        <label class="form-field">
+          How old are You?
+          <input
+          type="number"
+          name="age"
+          min="1"
+          max="100"
+          value="12">
```

Comment:
```suggestion
          <input
            type="number"
            name="age"
            min="1"
            max="100"
            value="12"
          >
```
wrong formatting, check indents